files/gitignore

```diff
@@ -9,6 +9,7 @@ node_modules/
 # misc
 /.env*
 /.pnp*
+/.pnpm-debug.log
```

Comment:
Shouldn't the package manager entries be applied conditionally as well? Seems cleaner to me.
I think we also need an `/pnpm-lock.ember-try.yaml` for `ember-try`?

Unrelated to this PR, are the following entries still relevant?
- /.sass-cache
- /connect.lock
- /libpeerconnection.log

Reply:
@bertdeblock I've followed what we already have in https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/gitignore (which subsequently used by addon blueprint, if I'm not mistaken), where we have both yarn and npm files and don't have pnpm yet.

regarding other entries: tbh, never saw those files around so don't know where they come from and if it's still relevant today.
anyways I think this should be in sync with ember-cli

Reply:
I think we can update those files in ember-cli as well, made [an issue](https://github.com/ember-cli/ember-cli/issues/10063) for it some days ago.
That's of course unrelated to this PR as well.
I'm just not really a big fan of adding entries to these ignore files if they are not needed I think.
We apply various things in different files based on the chosen package manager, so I don't really understand why we don't do the same for the ignore files.

Reply:
> We apply various things in different files based on the chosen package manager, so I don't really understand why we don't do the same for the ignore files.

fair enough.

In order to keep PR focused, I suggest leave it as is and I can create a follow up PR to make entries npmignore/gitgnore conditional based on the yarn/npm/pnpm flag